### DISCUSSION TITLE
DAI 2024-2025 - My picocli project - Aude Laydu

### DISCUSSION THREAD

**Discussion by eau2** [heig-vd-dai-course/heig-vd-dai-course#345] [Show and tell]
https://github.com/eau2/java-intellij-idea-and-maven

**Comment by ludelafo** [comment]
Everything seems fine for me except the `pom.xml`. It seems you have removed the initial properties from the following lines:

```xml
<properties>
    <!-- Omitted for brevity -->
</properties>
```

You must keep the initial values to ensure Maven can encode the files correctly (more on this in the next course) and check for the right Java version.

Also, you must only keep the `maven-shade-plugin`. The `maven-jar-plugin` can be removed to avoid any conflicts between the two packaging plugins. :)
